packages/snaps-rpc-methods/src/restricted/getBip32Entropy.ts

```diff
@@ -115,6 +129,7 @@ export function getBip32EntropyImplementation({
       curve: params.curve,
       path: params.path,
       secretRecoveryPhrase: await getMnemonic(),
+      cryptographicFunctions: getClientCryptography(),
```

Comment:
Should we add a test to each of the RPC methods that the crypto functions are passed correctly?